packages/cactus-plugin-ledger-connector-besu/src/main/typescript/api-client/besu-api-client.ts

```diff
@@ -23,6 +23,7 @@ export class BesuApiClient extends DefaultApi {
   private readonly log: Logger;
   private readonly wsApiHost: string;
   private readonly wsApiPath: string;
+  getPrometheusExporterMetricsV1: any;
```

Comment:
@Leeyoungone Is this some leftover change that is otherwise not necessary? Just asking because it doesn't seem to be used in the code of the class.

Reply:
I think you're right, I don't remember having to alter the besu-api-client.ts file, thank you!

Reply:
@Leeyoungone No worries! If you remove the line instead of commenting it that the file will nicely disappear from the diff altogether (and people won't wonder what the purpose of that commented code might be!). :-)

Once that's done, please squash + rebase and then it's LGTM